eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -204,8 +204,9 @@ stages:
       - template: tools/clone-buildtools/clone-buildtools.yml@azure-sdk-build-tools
       - download: current
         artifact: ${{parameters.ArtifactName}}-signed
-      - task: MSBuild@1
-        displayName: 'Publish to blobfeed'
+      - task: NuGetCommand@2
```

Comment:
While you are in here can you also add some logic that will change the DevoOps feed to use the azure-sdk-for-net-pr feed when building from the azure-sdk-for-net-pr repo? We need to be sure we don't publish packages publicly from that repo.

Reply:
I added the Id to the feeds as variables in global.yml

Reply:
Cool. Please add a condition to this task that checks that DevOpsFeedId isn't empty and if it is empty it will skip it.